src/main/java/nl/tudelft/ti2206/group9/gui/MainMenuScene.java

```diff
@@ -64,9 +73,10 @@
         settingsButton.setTooltip(new Tooltip("Change game settings"));
         loadButton.setTooltip(new Tooltip("Continue a game"));
         INPUT.setTooltip(new Tooltip("Enter your name"));
```

Comment:
That is because 'space' is being used instead of 'tab'. This might have to do with IntelliJ.
We will have a look at this at the end of the week anyway.